### DISCUSSION TITLE
Net 6.0

### DISCUSSION THREAD

**Discussion by atari-monk** [omansak/libvideo#233] [Q&A]
Hello
I tried to use it in .net6.0 projects.
It won't work in vscode.
There wont be .net 6.0 version ?

**Comment by omansak** [comment]
Thank you for feedback, i will check that

**Comment by ranjit5544** [comment]
You can use .Net 6.0. First you need to install .Net 6.0 on your system it will list.

**Reply by atari-monk** [reply]
I have .net 6.0. With Visual Studio 2022 it compiles ok.
But with Visual Studio Code and in dotnet CLI i have errors.
If i remeber correctly it misses net standard, but why do i need that i have .net dev packege installed.
All .net 6.0 projects i run are fine.
